dask/tests/test_base.py

```diff
@@ -871,15 +872,19 @@ def get(dsk, keys, *args, **kwargs):
     assert called[0]
 
 
-@pytest.mark.skipif('not da')
 @pytest.mark.parametrize('num_workers', range(1, 4))
 @pytest.mark.parametrize('scheduler', ['threads', 'processes'])
 def test_num_workers_config(num_workers, scheduler):
     # Regression test for issue #4082
-    a = da.random.randint(0, 10, size=400, chunks=2)
 
+    @delayed
+    def f(x):
+        time.sleep(0.5)
+        return x
+
+    a = [f(i) for i in range(5)]
     with dask.config.set(num_workers=num_workers), Profiler() as prof:
-        a.compute(scheduler=scheduler)
+        a = compute(*a, scheduler=scheduler)
```

Comment:
Given the increased time I recommend that we remove the `num_workers` parametrization, and instead do this once for an atypical value, like 3.